deployment/sriov-network-operator/values.yaml

```diff
@@ -95,6 +95,7 @@ images:
   sriovConfigDaemon: ghcr.io/k8snetworkplumbingwg/sriov-network-operator-config-daemon
   sriovCni: ghcr.io/k8snetworkplumbingwg/sriov-cni
   ibSriovCni: ghcr.io/k8snetworkplumbingwg/ib-sriov-cni
+  ovsCni: quay.io/kubevirt/ovs-cni-plugin
```

Comment:
ghcr.io/k8snetworkplumbingwg/ovs-cni-plugin

could we migrate to host image here instead of quay?

Reply:
https://github.com/k8snetworkplumbingwg/ovs-cni

or am I mixing these up? this is the proj correct?

Reply:
@Eoghan1232 
Yes, the repo is in k8snetworkplumbingwg, but it is managed by kubevirt CI. I didn't find ovs-cni image in github registry, I think it is only uploaded kubevirt registry